pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/render/LatexDocumentRenderer.kt

```diff
@@ -117,7 +124,7 @@ object LatexDocumentRenderer : DocumentRenderer<LatexDocument> {
         appendNewCmd("saksinfomottaker") {
             appenCmd("begin", "saksinfotable", "")
             verge?.let {
-                appendln("""\feltvergenavnprefix & \feltvergenavn \\""", escape = false)
+                appendln("""\felt${LanguageSetting.Sakspart.vergenavn} & \feltvergenavn \\""", escape = false)
```

Comment:
Bruk languagesetting for feltvergenavn også

Reply:
Obs